code/modules/projectiles/guns/energy/tetra.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/gun/energy/tetra
```

Comment:
This is going to need the new gun number things

Reply:
done